### PR TITLE
Run Helm tests in pipeline

### PR DESCRIPTION
**Pull request by JNygaard-Skylight**

### CODE REVIEW

File: .github/workflows/test.yaml [hunk 1]
```diff
@@ -21,3 +21,33 @@ jobs:
...
+            uses: azure/setup-kubectl@v1
+            with:
+              version: 'v1.20.0'
+
+          - name: Setup Helm
+            uses: azure/setup-helm@v1
+            with:
+              version: 'v3.4.0'
+
+          - uses: actions/setup-python@v4
```
Comment (emmastephenson) [inline .github/workflows/test.yaml:40]: Is the Python a dependency for the chart testing/cluster?

File: charts/alerts/templates/deployment.yaml [hunk 2]
```diff
@@ -1,38 +1,38 @@
...
-        app: alerts-app
+        app: {{.Values.appName}}
     spec:
       imagePullSecrets:
       containers:
-        - name: {{ .Chart.Name }}
+        - name: {{.Chart.Name}}
           image: ghcr.io/cdcgov/phdi/alerts:{{ .Values.image.tag }}
-          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          imagePullPolicy: {{.Values.image.pullPolic}}
```
Comment (emmastephenson) [inline charts/alerts/templates/deployment.yaml:22]:
```suggestion
          imagePullPolicy: {{.Values.image.pullPolicy}}
```

**Review by emmastephenson — CHANGES_REQUESTED** [review]
This looks great, thanks Josh! Just a few minor comments.

One note - I did see a couple of linting errors on `yaml` files, do you have [pre-commit enabled](https://github.com/CDCgov/phdi/wiki/Local-Development#formatting--pre-commit-hooks) for this repo? It'll autoformat your yaml files when you commit.

File: .github/workflows/test.yaml (hunk 1 above)
Comment (JNygaard-Skylight) [reply .github/workflows/test.yaml:40]: Yes that is correct.

File: charts/ingress/templates/ingress.yaml [hunk 3]
```diff
@@ -9,11 +9,10 @@ metadata:
         cert-manager.io/cluster-issuer: letsencrypt-prod
 spec:
     tls:
-        - hosts:
-              - {{.Values.ingressHostname: null}: null}
```
Comment (emmastephenson) [inline charts/ingress/templates/ingress.yaml:13]: Thanks for cleaning these up! My `pre-commit` added them the other day 🙃

**Review by emmastephenson — CHANGES_REQUESTED** [review]
This is fantastic, thanks Josh! Just one thing left before you merge - the version # of all the charts will need to be updated so that a new release is triggered. But otherwise I'm excited to see this merged! 🎉

File: charts/alerts/Chart.yaml [hunk 4]
```diff
@@ -1,6 +1,7 @@
+---
 apiVersion: v2
 name: alerts-chart
 description: A Helm chart for Kubernetes
 type: application
 version: 0.1.6
-appVersion: "1.0.8"
+appVersion: 1.0.8
```
[the remark this answers is not in the document]
Comment (JNygaard-Skylight) [reply charts/alerts/Chart.yaml]: I bumped all of the chart's minor version numbers. I changed how the chart names and labels are defined, using variables in each chart's `values.yaml`, so I considered changing the major version, but none of the names and labels should have different values

**Review by emmastephenson — APPROVED** [review]
LGTM! Thanks for this Josh! 🎉

**Pull request merged**